pkg/jobrunaggregator/jobruninstallanalyzer/analyzer.go

```diff
@@ -0,0 +1,305 @@
+package jobruninstallanalyzer
+
+import (
+	"context"
+	"encoding/xml"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"cloud.google.com/go/storage"
+
+	"k8s.io/apimachinery/pkg/util/clock"
+	prowv1 "k8s.io/test-infra/prow/apis/prowjobs/v1"
+
+	"github.com/openshift/ci-tools/pkg/jobrunaggregator/jobrunaggregatorapi"
+	"github.com/openshift/ci-tools/pkg/jobrunaggregator/jobrunaggregatorlib"
+	"github.com/openshift/ci-tools/pkg/junit"
+)
+
+// JobRunInstallAnalyzerOptions
+// 1. get a list of jobs from big query that meet the passed criteria: platform, network etc.
+// 2. finds jobruns for matching jobs for the specified payload tag
+// 3. reads all junit for the each jobrun
+// 4. constructs a synthentic junit that check minimum required succuessful installations
+type JobRunInstallAnalyzerOptions struct {
+	payloadTag               string
+	workingDir               string
+	jobRunStartEstimate      time.Time
+	clock                    clock.Clock
+	timeout                  time.Duration
+	platform                 string
+	infrastructure           string
+	network                  string
+	ciDataClient             jobrunaggregatorlib.CIDataClient
+	ciGCSClient              jobrunaggregatorlib.CIGCSClient
+	gcsClient                *storage.Client
+	minimumSuccessfulInstall int
+}
+
+func (o *JobRunInstallAnalyzerOptions) getRelatedJobRuns(ctx context.Context) ([]jobrunaggregatorapi.JobRunInfo, error) {
+	var jobRunsToReturn []jobrunaggregatorapi.JobRunInfo
+	jobs, err := o.ciDataClient.ListAllJobs(ctx)
```

Comment:
I don't see how you will be able to invert in this direction when this is run via a /payload command.

You need to be able to inspect a list of JobNames (release controller knows them all and so does the payload command) and select which ones match.  This means adding a `--jobNames=job1,job2,job3,job4` etc